mapper-runtime/src/main/java/com/datastax/oss/driver/api/mapper/DefaultRunTimeAttributes.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.mapper;
+
+import com.datastax.oss.driver.api.core.ConsistencyLevel;
+import com.datastax.oss.driver.api.core.config.DriverExecutionProfile;
+import com.datastax.oss.driver.api.core.metadata.Node;
+import java.nio.ByteBuffer;
+import java.time.Duration;
+import java.util.Map;
+
+public class DefaultRunTimeAttributes implements RuntimeAttributes {
+
+  private DriverExecutionProfile profile = null;
+
+  private boolean isIdempotent = false;
+
+  private boolean isIdempotentSet = false;
+
+  private int pageSize = -1;
+
+  private ConsistencyLevel consitencyLevel = null;
+
+  private ConsistencyLevel serialConsistencyLevel = null;
+
+  private Duration timeout = null;
+
+  private ByteBuffer routingKey = null;
+
+  private String routingKeyspace = null;
+
+  private boolean tracing = false;
+
+  private boolean tracingSet = false;
+
+  private ByteBuffer pagingState = null;
+
+  private long timeStamp = -1;
```

Comment:
So far in the driver codebase we don't capitalize "stamp" (e.g. `Statement.getQueryTimestamp()`).